api/scripts/clean-anonymized-users-passwords.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+require('dotenv').config();
+
+const _ = require('lodash');
+const bluebird = require('bluebird');
+
+const { parseCsvWithHeader } = require('./helpers/csvHelpers');
+const authenticationMethodRepository = require('../lib/infrastructure/repositories/authentication-method-repository');
+const AuthenticationMethod = require('../lib/domain/models/AuthenticationMethod');
+
+async function cleanAnonymizedUsersPasswords({ arrayOfAnonymizedUsersIds }) {
+  const anonymizedUserIdsWithPasswordDeleted = [];
+  await bluebird.mapSeries(arrayOfAnonymizedUsersIds, async (userId) => {
+    const numberOfRowDeleted = await authenticationMethodRepository.removeAllAuthenticationMethodsByUserId({
+      userId,
+      identityProvider: AuthenticationMethod.identityProviders.PIX,
+    });
+    if (numberOfRowDeleted > 0) {
+      anonymizedUserIdsWithPasswordDeleted.push(userId);
+    }
+  });
+  return anonymizedUserIdsWithPasswordDeleted;
+}
+
+async function main() {
+  console.log('Starting cleaning anonymized users with passwords');
```

Comment:
```suggestion
  console.log('Starting deleting anonymized users authentication methods');
```

Je sais pas si c'est correct mais c'est plus l'idée.